packages/react-variant-view/package.json

```
@@ -27,7 +27,7 @@
     "tcm": "tcm -p src/**/*.module.scss",
     "tcm:watch": "yarn run tcm --watch",
     "prepare": "yarn run build",
-    "test": "cross-env CI=1 jest --env=jsdom --runInBand --passWithNoTests",
+    "test": "cross-env CI=1 jest $GREP --env=jsdom --runInBand --passWithNoTests",
```

Why don't we use the same script here? Is it because this package has no test cases? 
```suggestion
    "test": "cross-env jest $GREP --env=jsdom --runInBand --ci --reporters=default --reporters=jest-junit --passWithNoTests",
```

---

oh, yeah, i didn't bother putting in packages that didn't have tests.  but i guess this one COUNT have tests.  so that's a good point.